/**
 * Запускает приложение
 * @returns {Promise<void>}
 */
const main = async () => {
  console.log("hello world!");
}


main()
    .catch(console.error);
